### PR TITLE
feat(3pnjp2h6): author card infomation

### PR DESCRIPTION
**Pull request by ngtrgiabao**
# What's this PR/MR do?
1. UI: Author Card Infomation

2. Function:

3. Impact range:

# Task link
- Example link

# TODO Checkboxes
- [x] Did you check UI is as same as Figma Design?
- [x] Did you check Spec/Flow?
- [x] Did you note influence scope + pull request link in sub task?
- [x] Did you check all test cases?
  1. Layout
  2. Responsive

# Screenshot (UI task)
| Description | Image |
| --- | --- |
| description_for_img | image |

### CODE REVIEW

File: components/common/CardInfo/index.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import {
```
Comment (hongsown) [inline components/common/CardInfo/index.tsx]: e tưởng dùng icon svg

File: components/common/CardInfo/index.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+      <div className="-mt-4">
+        <a href="/" className="flex items-end">
+          <div className="mr-2 aspect-square h-14 w-14 shrink-0 overflow-hidden rounded-full">
+            <Image src={avatar} alt="" width={56} height={56} loading="lazy" />
+          </div>
+          <span className="mt-5 font-bold">{name}</span>
+        </a>
+      </div>
+
+      <button type="button" name="button" className={cn(sx.FollowBtn)}>
```
Comment (hongsown) [inline components/common/CardInfo/index.tsx:52]: a Báu hôm qua bảo dùng AppButton

File: components/common/CardInfo/CardInfo.module.scss [hunk 3]
```diff
@@ -0,0 +1,11 @@
+.CardInfo {
+  @apply flex justify-center flex-col p-4 pt-0 gap-4 border-t-lg border border-white w-80 rounded-md;
+
+  & .FollowBtn {
```
Comment (tranduybau) [inline components/common/CardInfo/CardInfo.module.scss]:
```suggestion
   .FollowBtn {
```

File: components/common/CardInfo/index.tsx [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+interface IbodyCardInfo {
+  location?: string;
+  education?: string;
+  pronouns?: string;
+  workPosition?: string;
+  joined?: string;
+}
+
+const bodyCardInfo = ({
```
Comment (tranduybau) [inline components/common/CardInfo/index.tsx]:
```suggestion
const mappingBodyCardInfo/generateBodyCardInfo = ({
```

File: components/common/CardInfo/index.tsx [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+    description,
+    education,
+    name,
+    workPosition,
+    location,
+    pronouns,
+    className = "",
+  } = props;
+
+  const joined = new Date()
```
Comment (tranduybau) [inline components/common/CardInfo/index.tsx]: dayjs / date-fns

File: components/common/CardInfo/index.tsx [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+    return Object.entries(
+      bodyCardInfo({
+        location,
+        education,
+        pronouns,
+        workPosition,
+        joined,
+      }),
+    ).map(([key, value]) => (
+      <div key={key}>
```
Comment (tranduybau) [inline components/common/CardInfo/index.tsx]: Tách ra InformationItem

File: components/common/CardInfo/index.tsx [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+          <span>{value.content}</span>
+        </div>
+      </div>
+    ));
+  };
+
+  return (
+    <div className={cn(sx.CardInfo, className)}>
+      <div className="-mt-4">
+        <a href="/" className="flex items-end">
```
Comment (tranduybau) [inline components/common/CardInfo/index.tsx]:
```suggestion
        <AppLink href="/" className="flex items-end">
```

File: components/common/CardInfo/index.tsx [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+  } = props;
+
+  const joined = new Date()
+    .toISOString()
+    .split("T")[0]
+    .split("-")
+    .reverse()
+    .join("/");
+
+  const renderBodyCard = () => {
```
Comment (tranduybau) [inline components/common/CardInfo/index.tsx]:
```suggestion
  const RenderBodyCard = () => {
```

**Review by tranduybau — APPROVED** [review]

**Pull request merged**